packages/editor/src/components/post-card-panel/index.js

```diff
@@ -94,10 +95,7 @@ export default function PostCardPanel( { className, actions, children } ) {
 							) }
 						</VStack>
 					) }
-					{
-						// Todo: move TemplateAreas (and the selectors it depends) to the editor package, and use it here removing the children prop.
-						children
-					}
+					<TemplateAreas />
```

Comment:
Should we limit this to the "template" post type for now?

Reply:
Hi @youknowriad the check was added.